src/commands/GameIntegration/overwatch.ts

```diff
@@ -73,7 +73,7 @@ export default class OverwatchCommand extends SkyraCommand {
 				].join('\n'))
 				.addField(EMBED_DATA.RATINGS_TITLE, EMBED_DATA.RATINGS([
 					...ratings.values(),
-					{ role: 'average', level: overwatchData.rating }
+					{ role: 'average', level: overwatchData.rating === 0 ? EMBED_DATA.NO_AVERAGE : overwatchData.rating }
```

Comment:
Shouldn't this be checking the array's length?
```suggestion
					{ role: 'average', level: overwatchData.rating?.length ? EMBED_DATA.NO_AVERAGE : overwatchData.rating }
```

Reply:
No, `rating` is a number (on the API data). `ratings` is an array (parsed from the API data)